satellite/metainfo/config.go

```diff
@@ -140,17 +140,17 @@ type PointerDB interface {
 }
 
 // OpenStore returns database for storing pointer data.
-func OpenStore(ctx context.Context, logger *zap.Logger, dbURLString string) (db PointerDB, err error) {
+func OpenStore(ctx context.Context, logger *zap.Logger, dbURLString string, app string) (db PointerDB, err error) {
```

Comment:
I think the application_string should be inside the dbURLstring already rather than pass it as a separate argument, but I guess this is fine as well.